src/main_dpdk.cpp

```diff
@@ -412,6 +414,8 @@ static int COLD_FUNC  usage() {
     printf("                              argument from config file. Supported method are 1 for nanoseconds (clock_gettime), 0 (default) for standard system ticks (RDTSC)\n");
     printf(" --timesync-method <method> : Enable time synchronisation with given method. Overrides the 'timesync_method' argument from config file\n");
     printf("                              Supported method is 1 for PTP. Default is 0 for no synchronisation\n");
+    printf(" --timesync-transport <method> : Set transport method for PTP packets. Overrides the 'timesync_method' argument from config file\n");
+    printf("                              Supported method is 'eth' and 'udp'. Default is 'eth' if timesync-method was set to PTP\n");
```

Comment:
Wrong help text, should be “… Overrides the 'timesync_transport' argument…”  Also write about possible values: 1 for UDP, 0 for raw L2/Ethernet (?)